.github/workflows/ccm.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches: 
       - main
       - '[0-9][0-9][0-9][0-9].[0-9][0-9].*' # 2021.01.x
+      - '2024-03-01-dep-update'
     tags:
```

Comment:
I don't think this change should be part of the PR. since I fee the rebase has some problem